rundeckapp/grails-app/views/framework/_pluginConfigPropertyFormField.gsp

```diff
@@ -268,6 +268,7 @@
         <g:set var="syntaxSelectable" value="${prop.renderingOptions?.(StringRenderingConstants.CODE_SYNTAX_SELECTABLE)}"/>
         <g:set var="syntaxModes" value="${prop.renderingOptions?.(StringRenderingConstants.CODE_SYNTAX_MODES_ALLOWED)}"/>
 
+        <g:set var="extraInputCss" value="${extraInputCss?.replaceAll("context_var_autocomplete","")}"/>
```

Comment:
i think the only usage for `extraInputCss` is to set it to `context_var_autocomplete` currently, so could probably just remove it from the handling for CODE display type, i.e. remove it from line 284

Reply:
I did that first but then I changed it hehhe